mlfinlab/feature_importance/cfi.py

```diff
@@ -0,0 +1,113 @@
+"""
+An Implementation of the Clustered Feature Importance algorithm described by
+Dr Marcos Lopez de Prado in 'Clustered Feature Importance (Presentation Slides)'
+(https://papers.ssrn.com/sol3/papers.cfm?abstract_id=3517595)
+"""
+
+#Imports
+import numpy as np
+import pandas as pd
+import mlfinlab as ml
+from sklearn.metrics import log_loss
+from scipy.cluster.hierarchy import linkage, fcluster
+from information_matrix import  get_dependence_matrix , get_distance_matrix
+
+
+def cluster_features(X : pd.DataFrame , linkage_method : str , n_clusters : int = None ):
+    '''
+    Get Clusters containing features subsets
+
+    (param) X : (pd.DataFrame) of features
```

Comment:
This doc string looks a bit strange. 

1. has `(param)`
2. check argument names